### ISSUE TITLE
Once typed, replace the same placeholder everywhere within a snippet

### ISSUE THREAD

**Issue by AHThomas** [GaurangTandon/ProKeys#194]
**Steps to reproduce issue**
Doing the following in latest ProKeys version will cause the problem I'm writing about.

1. Create a snippet where you use the same placeholder multiple times, for example:
We saw your dog %patient% at our clinic today.  %patient% got a thorough exam and looks good.

2. Use the snippet, replace the first %patient% placeholder with a patient name such as "Rover"

**Actual behavior**
Only the first instance of %patient% is replaced with the text "Rover"

**Expected/desired behavior**
ALL instances of %patient% within the snippet would be replaced with whatever I enter for the first instance of that snippet, saving me from having to enter the same value N times within the same snippet.

-------

**Comment by GaurangTandon** [comment]
Thanks for your feedback! You're right. I have thought about this before as well. But, I could never get around to implementing as it requires reworking the existing system. I am sorry but I cannot give you a date as to when this will be implemented, at least not until May end.

**Comment by GaurangTandon** [comment]
closed as a duplicate of #131
